lib/flex/FlexController.js

```diff
@@ -60,7 +60,7 @@ class FlexController extends BaseController {
     this.projectSetup.save((err) => {
       if (err) {
         this.cliManager.log(LogLevel.WARN, 'Failed to save job ID to project settings.');
-        this.cliManager.log(LogLevel.ERROR, err);
+        this.cliManager.log(LogLevel.WARN, err);
```

Comment:
What is the second argument of the log() function? Here you supply an error object and on the previous line you supply an error message. Is this OK? Can this be done using a single call to log()?

Reply:
Merged in one call.